app/js/disease-associations-controller.js

```diff
@@ -150,12 +164,123 @@ angular.module('cttvControllers')
     // on PAGE LOAD
     //
 
+    cttvUtils.clearErrors();
+    $scope.filters = cttvLocationState.getState()[facetsId] || {};
+    render(cttvLocationState.getState(), cttvLocationState.getOldState());
 
+    $scope.uploadedFile = function (element) {
+        $scope.$apply(function ($scope) {
+            $scope.files = element.files;
+        });
 
-    cttvUtils.clearErrors();
-    $scope.filters = cttvLocationState.getState()[facetsId] || {} ;
-    render( cttvLocationState.getState(), cttvLocationState.getOldState() );
+        $scope.addFile();
+    };
+
+    $scope.removeTargets = function(){
+        var theElement = document.getElementById("myFileInput");
```

Comment:
It is generally considered bad practice to access the DOM inside controllers. We are still probably doing it in a couple of places, but we tend to avoid doing it (and we should fix the places where we are doing it rather than adding more).
I suggest you create a directive for the whole thing and move all the relevant code inside that directive. That way in your markup you will only have something like:
<filter-by-target-list></filter-by-target-list>
and all the html template to deal with it as a template for the directive